### PR TITLE
Script to create tokens for flocking

### PR DESCRIPTION
**Pull request by matyasselmeci**
Runs automatically on startup and uses the pool password to generate tokens for all the hosts in FLOCK_FROM. Saves tokens to the usual place (`$HOME/.config/tokens.d`) and also prints them to stdout. When run manually, you can use a different key.

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Don't merge this yet. Need to change something.

File: Dockerfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ RUN \
 
 COPY supervisord.conf /etc/supervisord.conf
 COPY condor/*.conf /etc/condor/config.d/
-COPY start.sh update-config update-secrets /
+COPY start.sh update-config update-secrets create-flocking-tokens /
 COPY fetch-crl.cron /etc/cron.d/fetch-crl
 
 CMD ["/bin/bash", "-x", "/start.sh"]
```
Comment (brianhlin) [inline Dockerfile:37]: Side note: you should just use `supervisord_startup.sh` and `/etc/osg/image-config.d/`

**Review by brianhlin — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (matyasselmeci) [reply Dockerfile:37]: I'll add that to the 'technical debt todo list'

**Pull request merged**